demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -599,6 +602,114 @@ def _get_invalid_dbot_outputs(context_outputs_paths, context_outputs_description
 
         return missing_outputs, missing_descriptions
 
+    def check_all_reputation_outputs(
+        self, command_name: str, outputs: Set[str], used_reputation_objects: Set[str]
+    ) -> bool:
+        """check if all mandatory outputs for custom object do exist.
+        Args:
+            command_name: the command name
+            outputs: outputs of the command
+            used_reputation_objects: reputation objects (like url, ip, etc) used in the command's context outputs.
+        Returns:
+            bool: Whether all mandatory outputs of custom outputs exist or not
+        """
+        missing_outputs = []
+        objects_missing_outputs = set()
+        outputs_exist = True
+
+        for ioc in used_reputation_objects:
+            for output in IOC_OUTPUTS_DICT.get(ioc.lower(), ()):
+                if output not in outputs:
+                    missing_outputs.append(output)
+                    objects_missing_outputs.add(ioc)
+
+        if missing_outputs:
+            error_message, error_code = Errors.command_reputation_output_is_missing(
+                command_name,
+                missing_outputs,
+                objects_missing_outputs,
+                XSOAR_CONTEXT_STANDARD_URL,
+            )
+            if self.handle_error(
+                error_message,
+                error_code,
+                file_path=self.file_path,
+                warning=self.structure_validator.quiet_bc,
+            ):
+                outputs_exist = False
+        return outputs_exist
+
+    @staticmethod
+    def get_outputs(command: dict) -> Tuple[set, list, set, set]:
```

Comment:
too many outputs. Why is this needed? It may be difficult for future maintainers to understand the order. 

To return more than two value, use a `NamedTuple`. 
Please take it with Danny, I doubt this is necessary.